app/src/org/commcare/dalvik/activities/CallOutActivity.java

```diff
@@ -107,16 +107,17 @@ public void onCancel(DialogInterface dialog) {
     }
     
     private void dispatchAction(String action) {
+        // using createChooser to handle any errors gracefully
         if(Intent.ACTION_CALL.equals(action) ) {
             tManager.listen(listener, PhoneStateListener.LISTEN_CALL_STATE);
             
             Intent call = new Intent(Intent.ACTION_CALL);
             call.setData(Uri.parse("tel:" + number));
-            startActivity(call);
+            startActivity(Intent.createChooser(call, "Dialing " + number));

```

Comment:
Two comments

1) Does this create a chooser if there's only one dialer on the phone? That's probably confusing to end users who have to provide input that there's only one choice for.

2) We should make sure this is localized (pulled from translation files) if we are going to use this approach